test/test_torch.py

```diff
@@ -9659,6 +9659,21 @@ def add_neg_dim_tests():
         assert not hasattr(TestTorch, test_name), "Duplicated test name: " + test_name
         setattr(TestTorch, test_name, make_neg_dim_test(name, tensor_arg, arg_constr, types, extra_dim))
 
+class TestRPATH(TestCase):
+    @unittest.skipIf(not sys.platform.startswith('linux'), "linux-only test")
+    def test_rpath(self):
+        """
+        Make sure RPATH (or RUNPATH) in nvrtc does not contain a cuda stubs directory
+        issue gh-35418
+        """
+        libdir = os.path.join(os.path.dirname(torch._C.__file__), 'lib')
+        caffe2_nvrtc = os.path.join(libdir, 'libcaffe2_nvrtc.so')
+        if os.path.exists(caffe2_nvrtc):
```

Comment:
```suggestion
        self.assertTrue(os.path.exists(caffe2_nvrtc))
```

Reply:
See above: Makes only sense if the skip guard checks if PyTorch is built with CUDA support. Not sure if `TEST_CUDA`, i.e. `torch.cuda.is_available()` is correct here or if that does check if CUDA is **currently** available. I fear it might miss the exact use case here: When running on a machine without CUDA drivers but only CUDA stub libs which cannot be loaded/used.